plantseg/segmentation/gasp.py

```diff
@@ -1,14 +1,24 @@
 import time
+from functools import partial
 
 import numpy as np
-from GASP.segmentation import GaspFromAffinities, WatershedOnDistanceTransformFromAffinities
+from GASP.segmentation import GaspFromAffinities
 from GASP.segmentation.watershed import SizeThreshAndGrowWithWS
 
 from plantseg.pipeline import gui_logger
 from plantseg.pipeline.steps import AbstractSegmentationStep
+from plantseg.segmentation.dtws import compute_distance_transfrom_watershed
 from plantseg.segmentation.utils import shift_affinities
 
 
+class WSSegmentationFeeder:
```

Comment:
This class is used just in one place and can easily be replaced by the lambda expression `lambda  _: segmentation`, so no need for additional boiler plate